usb.c

```diff
@@ -193,8 +193,10 @@ int qdl_read(struct qdl_device *qdl, void *buf, size_t len, unsigned int timeout
 	int ret;
 
 	ret = libusb_bulk_transfer(qdl->usb_handle, qdl->in_ep, buf, len, &actual, timeout);
-	if (ret < 0)
+	if (ret < 0){
+		warnx("bulk read failed: %s", libusb_strerror(ret));
```

Comment:
Will this not result in two timeout prints when a timeout happens?
And for the initial firehose_read() in firehose_run(), we do expect to get a timeout once we've consumed the LOG entries. I would hope we can remove the existing "firehose operation timed out" print.